examples/complete/variables.tf

```
@@ -103,3 +103,33 @@ variable "allowed_cidr_blocks" {
     "0.0.0.0/0",
   ]
 }
+
+variable "root_block_device_encrypted" {
+  type        = bool
+  default     = false
+  description = "Whether to encrypt to root block device"
+}
+
+variable "root_block_device_volume_size" {
+  type        = number
+  default     = 8
+  description = "The volume size (in GiB) to provision for the root block device. It cannot be smaller than the AMI it refers to."
+}
+
+variable "metadata_http_endpoint" {
```

```suggestion
variable "metadata_http_endpoint_enabled" {
```

we have the convention to add `_enabled` suffix to all boolean flags